app/models/event.rb

```diff
@@ -21,4 +25,25 @@ def upcoming_agenda_dates
     def upcoming_events
       Event.this_week_forward.sort_by(&:occurs_at)
     end
+
+    # TODO: Here is an example of a location that is not at the capitol complex:
+    #   {:field=> {
+    #     :"building-name"=>"Alpena Community College - Granum Theatre",
+    #     :"street-address"=>"665 Johnson Street",
+    #     :city=>"Alpena",
+    #     :state=>[
+    #       {:"postal-code"=>"MI"},
+    #       {:"state-fullname"=>"Michigan"}
+    #     ],
+    #     :zip=>"49707"}}
+    # For now, events that are not at the capitol complex are not formated and
+    # cached, so their `location` will be empty.
+    def cache_formatted_location
+      building_name = raw_location[:'capitol-complex'][:building]
+      room_name = raw_location[:'capitol-complex'][:room]
+      self.location = "Building: #{building_name}, Room: #{room_name}"
```

Comment:
Later, do you think it would be useful for us to build a look up table/method for these office locations so when we generate ICS files or sync to Google calendar, we can provide location information? I'm guessing most people in the area know these locations but once you include real physical location info you get distance and departure notifications for free on some phones/clients.
 
For example, RHOB is the Rayburn House Office Building, 45 Independence Ave SW, Washington, DC 20515.

If so, this might be a good secondary issue to create